samples/common/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+    id("kotlin-parcelize")
+    id("appyx-publish-android")
+    id("appyx-lint")
+    id("appyx-detekt")
+}
+
+android {
+    namespace = "com.bumble.appyx.common"
```

Comment:
`com.bumble.appyx.sample.common` - otherwise this could get confusing in the future if we ever create another common module.